CONTRIBUTING.rst

```diff
@@ -257,3 +257,27 @@ Below is a list of pure community features, their community maintainers, and mai
 * Description:
 
 ``Enable boot of the installed system from a BTRFS subvolume.``
+
+Code maintainers
+----------------
+
+This section will provide you information about a contact person who is responsible for part of the 
+Anaconda code-base and or related projects. Anaconda team also use this list to avoid situation
+when does not have main maintainer or backup.
+
+Backup of the code should be familiar and develop parts similar way as maintainer does.
+Responsibility of a backup is to check all PRs created to this part of the Anaconda.
+
+=================== ========== ======
+Code part           maintainer backup
+=================== ========== ======
+Network             rvykydal
+Storage             vponcova
+Payload             jkonecny
+Subscription        mkolman
+dbus                vponcova
+configuration files vponcova
+dasbus              vponcova
+simpleline          jkonecny
+initial-setup       mkolman
+=================== ========== ======
```

Comment:
I think we're missing a lot of things here. Particularly pykickstart and Blivet. While these are technically not Anaconda, they are pretty much linked with everything we do. (I don't buy the "different red hat team" argument. That won't do any good to anyone reading this)

I'm not sure if kickstart tests should have an entry here, too?

Reply:
Maybe also missing: networked storage, packaging & releases,...

Reply:
Updated